examples/k8s/advanced/1-create-cluster.sh

```diff
@@ -3,7 +3,7 @@ set -o errexit
 
 # create registry container unless it already exists
 reg_name='kind-registry'
-reg_port='5000'
+reg_port='8049'
```

Comment:
Port 5000 is reserved for the ControlCenter system service on macOS, so the example fails because it tries to bind to that port. I actually didn't intend to include that in this changeset, but maybe I will just keep it for other Mac users. Lmk if you'd prefer I move it to a separate PR.

Reply:
That's fine. I think it's ok to change.